code/modules/admin/topic.dm

```diff
@@ -356,9 +356,18 @@
 		if(!check_rights(R_SERVER))
 			return
 
-		emergency_shuttle.settimeleft( input("Enter new shuttle duration (seconds):","Edit Shuttle Timeleft", emergency_shuttle.timeleft() ) as num )
+		var/new_timeleft = input("Enter new shuttle duration (seconds):","Edit Shuttle Timeleft", emergency_shuttle.timeleft() ) as num | null
+		if(!new_timeleft)
+			return
+
+		if( alert("Do you want this to be announced?",,"Yes","No" ) == "Yes" )
+			if(new_timeleft < emergency_shuttle.timeleft())
+				captain_announce("The emergency shuttle found a shortcut. It will arrive in [round(new_timeleft/60)] minutes.")
```

Comment:
Could you come up with something a little more logical for a spacefaring vessel?

Even as simple as
"is arriving ahead of schedule"
"hit the turbo"
"has been scrambled for faster deployment"

Reply:
"has engaged nitro afterburners" is clearly the best option here.

Reply:
keep them coming, I'm adding all of them